src/actions/index.js

```diff
@@ -0,0 +1,19 @@
+import { SIGNUP_REQUEST, SIGNUP_SUCCESS } from "./types";
+
+export const signUp = userDetails => async dispatch => {
+  dispatch({ type: SIGNUP_REQUEST, payload: userDetails });
+
+  const res = await fetch(
```

Comment:
create a service file which can be used to call api's

Reply:
use promise's instead of await

Reply:
Okay sir